code/rdb/datastripe.q

```diff
@@ -10,7 +10,9 @@
     .lg.o[`reload;"Kept ",string[.ds.periodstokeep]," period",$[.ds.periodstokeep>1;"s";""]," of data from : ",", " sv string[tabs]];
 
     // update the access table in the rdb
-    .rdb.access:update start:lasttime^(.ds.getstarttime each key .rdb.tablekeycols), stptime:data[][`time] from .rdb.access;
+    t:tables[`.] except .rdb.ignorelist;
+    starttimes:.ds.getstarttime each t;
+    .rdb.access:update start:starttimes^start, stptime:data[][`time] from .rdb.access;
```

Comment:
`.rdb.access:update start:starttimes, stptime:data[][`time] from .rdb.access;`

otherwise this will not overwrite the old start column

Reply:
so does the not need to happen in the wdb? in the same place

Reply:
It gets overwritten in the rdb because data is cleared from memory each period. What is on disk isn't getting cleared so the start time stays the same.

Reply:
but this line in the wdb relates to the access table in memory which uses the in memory tables to calculate its times in the same way as in the rdb